.github/ISSUE_TEMPLATE.md

```diff
@@ -1,89 +1,68 @@
 <!--
-
-Please summarize the issue in the title, and then use the template below to
-fill out the details so we can reproduce the issue on our end.
-
+First and foremost, we’d like to thank you for taking the time to contribute to our project. Before submitting your issue, please follow these steps:
+
+1. Familiarize yourself with our contributing guide:
+	* https://github.com/desktop/desktop/blob/master/CONTRIBUTING.md#contributing-to-github-desktop
+2. Check if your issue is in the known-issues and work-arounds doc:
+	* https://github.com/desktop/desktop/blob/master/docs/known-issues.md
+3. Make sure your issue isn’t a duplicate of another issue
+4. If you have made it to this step, go ahead and fill out the template below
 -->
 
-### Description
-
-[Description of the issue]
-
-### Version
-
+## Description
 <!--
-
-What version of GitHub Desktop are you running? This is displayed under the
-`About GitHub Desktop` menu item. If you are running from source, include
-the commit by running `git rev-parse HEAD` from your local repository.
-
+Provide a detailed description of the behavior you're seeing or the behavior you'd like to see **below** this comment.
 -->
 
-**GitHub Desktop version:** [version here]
 
+## Version
 <!--
-
-The operating system you are running on may also help with reproducing the
-issue:
-
- - If you are on macOS, launch `About This Mac` and write down the OS version
-   listed.
- - If you are on Windows, open `Command Prompt` and attach the output of this
-   command: `cmd /c ver`
-
+Place the version of GitHub Desktop you have installed **below** this comment. What version of GitHub Desktop are you running? This is displayed under the 'About GitHub Desktop' menu ite,. If you are running from source, include the commit by running `git rev-parse HEAD` from the local repository.
```

Comment:
> menu ite**m**

Reply:
> ...under the 'About GitHub Desktop' menu ite,. If you are...

Looks like I had a dilemma between choosing a `,` and a `.`

🤦‍♂️